app-text/docbook-dsssl-stylesheets/docbook-dsssl-stylesheets-1.79-r3.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+# EAPI=6 is blocked by Gentoo bug 497052.
+EAPI=5
+
+MY_P=${P/-stylesheets/}
+
+inherit sgml-catalog
+
+DESCRIPTION="DSSSL Stylesheets for DocBook"
+HOMEPAGE="https://github.com/docbook/wiki/wiki"
+SRC_URI="mirror://sourceforge/docbook/${MY_P}.tar.bz2"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos ~sparc-solaris ~x64-solaris"
+IUSE=""
+
+RDEPEND="app-text/sgml-common"
+
+DOCS=(BUGS ChangeLog README RELEASE-NOTES.txt WhatsNew)
+
+S="${WORKDIR}/${MY_P}"
+
+src_unpack() {
+	unpack ${A}
+	cp "${FILESDIR}/${PN}-1.79.Makefile" "${S}/Makefile" || die
+}
+
+src_compile() { :; }
+
+src_install() {
+	emake \
+		BINDIR="${ED}usr/bin" \
+		DESTDIR="${ED}usr/share/sgml/docbook/dsssl-stylesheets-${PV}" \
+		install
+
+	dodoc "${DOCS[@]}"
+}
+
+sgml-catalog_cat_include \
```

Comment:
My logic: since catalog files are placed after src_install(), it's logical to add them to system-wide catalog after src_install(), not before. A matter of preference.